### DISCUSSION TITLE
Ban discussion board

### DISCUSSION THREAD

**Discussion by mgsweet** [mgsweet/Apex-NoRecoil-2021#294] [General]
This board is mainly used to:
1. Inform others that u got banned.
2. Ask whether others got banned.

For the first purpose, pls provide the following information:
- When did u get banned?
- What features did you use? 
- Do you also use any other doggy things?

**Comment by zzxqy** [comment]


**Reply by fatinghenji** [reply]
是否卡了最近的一系列bug

**Reply by zzxqy** [reply]
没有
